docs/en/stack/ml/anomaly-detection/ml-jobs-from-visuals.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[[ml-jobs-from-lens]]
+= Creating {anomaly-jobs} from Lens visualizations
+++++
+<titleabbrev>{anomaly-jobs-cap} from visualizations</titleabbrev>
+++++
+
+You can create {anomaly-jobs} from the compatible Lens charts on **Dashboard**. 
+
+[discrete]
+[[prereqs]]
+== Prerequisites
+
+* Only chart-like visualizations are compatible. Supported chart types are 
+`area`, `area_percentage_stacked`, `area_stacked`, `bar`, `bar_horizontal`, 
+`bar_horizontal_stacked`, `bar_percentage_stacked`, `bar_stacked`, and `line`.
+
+* The chart must contain a date field on one axis and it must be the same as the 
+default date field for the {data-source}.
+
+
+[discrete]
+[[create-job]]
+== Creating the job
+
+NOTE: You need to have a compatible visualization on **Dashboard** to create an 
+{anomaly-job}. If you don't have one but you want to try the feature out, go to 
+**Analytics > Dashboard** and select the `[Flight] Global Flight Dashboard` 
+which is based on the {kib} sample flight data set. Select the `Flight count` 
+visualization from the dashboard.
+
+
+. Go to **Analytics > Dashboard** and select a dashboard with a compatible 
+visualization.
+. Find the chart and select **More** from its **Options (...) menu**.
+. Select **Create {anomaly-job}**. The option is only displayed if the 
+visualization can be converted to an {anomaly-job} configuration.
+. (Optional) Select the layer from which the {anomaly-job} is created.
+
+[role="screenshot"]
+image::images/create-ad-job-from-lens.jpg[A screenshot of a chart with the Options menu opened]
+
+If the visualization has multiple compatible layers, you can select which layer 
+to use for creating the {anomaly-job}. 
+
+[role="screenshot"]
+image::images/select-layer-for-job.jpg[A screenshot of a chart with the Options menu opened]
+
+If multiple fields are added to the chart or you selected a `Break down by` 
+field, the multi metric job wizard is used for creating the job. For a single 
+metric chart, the single metric wizard is used.
+
+If the configured time range of the chart is relative, it is converted to 
+absolute start and end times in the job config. If the conversion of these times 
```

Comment:
I don't think it's necessary to shorten this word:
```suggestion
absolute start and end times in the job configuration. If the conversion of these times 
```

Reply:
🤦 You are absolutely right! Thanks for spotting this.